course/constants.py

```diff
@@ -368,6 +368,8 @@ class flow_permission:  # noqa
     set_roll_over_expiration_mode = "set_roll_over_expiration_mode"
     see_session_time = "see_session_time"
     lock_down_as_exam_session = "lock_down_as_exam_session"
+    send_interact_email_to_course_staff =\
+        "send_interact_email_to_course_staff"
```

Comment:
There should also be ones in the participation_permission(s), I will do that latter. The permission here is to restrict the usage of pages, I don't want to receive emails from students for all questions for all the time.